package/CHANGELOG

```diff
@@ -66,6 +66,8 @@ Fixes
     would create a test artifact (Issue #2979, PR #2981)
 
 Enhancements
+  * Preliminary support for the Linux ARM64 platform with minimal
+    dependencies (PR #2956)
```

Comment:
There's the orphaned issue #1989 here that could be added too if you felt like it.